src/xenia/kernel/xam/xam_avatar.h

```diff
@@ -0,0 +1,200 @@
+/**
+ ******************************************************************************
+ * Xenia : Xbox 360 Emulator Research Project                                 *
+ ******************************************************************************
+ * Copyright 2025 Ben Vanik. All rights reserved.                             *
+ * Released under the BSD license - see LICENSE in the root for more details. *
+ ******************************************************************************
+ */
+
+#ifndef XENIA_KERNEL_XAM_XAM_AVATAR_H_
+#define XENIA_KERNEL_XAM_XAM_AVATAR_H_
+
+#include "xenia/kernel/kernel_state.h"
+#include "xenia/xbox.h"
+
+namespace xe {
+namespace kernel {
+namespace xam {
+
+struct X_ASSET_ID {   // X_GUID
+  uint64_t data;      // 0x0 sz:0x8
+  uint32_t data2;     // 0x8 sz:0x4
+  uint32_t title_id;  // 0xC sz:0x4
+};
+static_assert_size(X_ASSET_ID, 0x10);
+
+// More Research Needed
+struct X_AVATAR_METADATA {
+  // body type exists between 0x120 and 0x130
+  uint8_t data1[0x4];
+  uint32_t weight;           // 0x4 sz:0x4
+  uint32_t height;           // 0x8 sz:0x4
+  uint8_t data3[0xF0];       // 0xC sz:0xF0
+  uint32_t skin_color;       // 0xFC sz:0x4
+  uint8_t data4[0x4];        // 0x100 sz:0x4
+  uint32_t lipstick_color;   // 0x104 sz:0x4
+  uint8_t data5[0x8];        // 0x108 sz:0x4
+  uint32_t eyeshadow_color;  // 0x110 sz:0x4
+  uint8_t data6[0x2D4];
+};
+static_assert_size(X_AVATAR_METADATA, 0x3E8);
+
+// https://github.com/hetelek/Velocity/blob/master/XboxInternals/AvatarAsset/AvatarAssetDefinintions.h
+enum X_AVATAR_BODY_TYPE : uint8_t { Unknown, Male, Female, All };
+
+// v2 assets its int32_t
+enum X_BINARY_ASSET_TYPE : uint32_t {
+  Component = 1,
+  Texture = 2,
+  ShapeOverride = 3,
+  Animation = 4,
+  ShapeOverridePost = 5,
+};
+
+enum X_ASSET_SUBCATEGORY : int32_t {
```

Comment:
This enum is once in hex other in decimal. decide if you use decimal or hexadecimal